src/Tickets/Flexible_Tickets/Base.php

```diff
@@ -508,4 +512,93 @@ public function render_series_details_for_attached_event( int $post_id ): void {
 			'action_links' => $action_links
 		] );
 	}
+
+	/**
+	 * Renders the series details on order report page for an event attached to a series.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $post_id The ID of the post being displayed.
+	 *
+	 * @return void
+	 */
+	public function render_series_details_for_attached_event_order_report( int $post_id ): void {
```

Comment:
It makes sense to do this in the `Base` controller, but it's growing too large.
Please extract this method, and others that might be about altering the reports, into a dedicated, proxied class (e.g. [like this](https://github.com/the-events-calendar/event-tickets/blob/7285785ea2a74359e84afdc42b351e6b5894e9a5/src/Tickets/Flexible_Tickets/Series_Passes.php#L100) and [this](https://github.com/the-events-calendar/event-tickets/blob/7285785ea2a74359e84afdc42b351e6b5894e9a5/src/Tickets/Flexible_Tickets/Series_Passes.php#L341)).